### PR TITLE
[UNTESTED CODE MAYBE IL DO IT TOMORROW]Makes radiation storms not as bullshit

### PR DESCRIPTION
**Pull request by silicons**
Makes radiation storms not as overpowered and bullshit. ;^):^)

They will have a random strength of 1 to 5.
That will be how many mutations/pulses you recieve.
It /should/ be equally spaced out.
You know, as in, not "5 radpulses every second ggnore anyone who's not a virologist with DAR or changeling"

### CODE REVIEW

**Comment by Banana-Fist** [comment]
rads op plz nerf.

**Comment by PKPenguin321** [comment]
Nice smileys kevinz XD

**Comment by Shadowlight213** [comment]
Wow. It's almost like you're supposed to go into maint? Maybe you should nerf ash storms too.
Have you tried getting good? :^)
Or you know, play plasmaman for dat rad immunity.
Rad storms are supposed to be powerful. You get a pretty big warning, and in the original implementation it auto activated emergency maint.

Also It's not ggnore. You're just stunned for the duration. Not once in my testing have I actually been put into crit from rad toxin damage.

**Comment by Dorsisdwarf** [comment]
Yeah, you get a big warning, and then you fucking die, spasticated and invisible, with no way to avoid it. Even the CE's hardsuit, which has 100%/90% coverage against rads will result in a near-instant fate worse than death. 

It's like if the alien event gave you a head's up, then just flat-out chest-burst anyone not in the surgery office.

**Comment by kevinz000** [comment]
Oh yeah I didn't test this code yet so I'm going to [DNM] for now because I'l do it tomorrow ;^)

**Comment by nullbear** [comment]
1-5 in total is too much of a nerf. But i agree that current radstorms are a tad too intense.

**Comment by bgobandit** [comment]
did anyone complain about radstorms, ever? shit is supposed to be deadly

**Comment by ma44** [comment]
If you were to increase the delay between the "haul ass to maintenance" and the station going into the storm, that would be also fine

**Comment by TechnoAlchemisto** [comment]
Events should be this disruptive and dangerous, this is one of the few events that actually forces you to fucking do something, don't nerf it. They aren't overpowered, just walk into fucking maint.

**Comment by PKPenguin321** [comment]
>If you were to increase the delay between the "haul ass to maintenance" and the station going into the storm, that would be also fine

Yeah this is definitely the better nerf

**Comment by KorPhaeron** [comment]
It needs an overlay so you can tell things are happening

**Comment by Shadowlight213** [comment]
True. I can increase the startup phase delay.
I might use the blob alarm sound or whatever for the winding up phase as well.
As for the overlay, I guess I can use the ash storm one, but recolor it green? Might look like crap though.

File: code/datums/weather/weather_types.dm [hunk 1]
```diff
@@ -134,20 +140,33 @@
...
-	L.rad_act(20,1)
+						if(prob(25))
+							randmuti(H)
+						if(prob(90))
+							randmutb(H)
+						else
+							randmutg(H)
+						H.domutcheck()
+		L.rad_act(20,1)
+		armed = 0
```
Comment (KorPhaeron) [inline code/datums/weather/weather_types.dm]: Mekhi this irradiates a single mob in the area and then goes on cooldown. If you had 10 assistants run into the storm only one would take damage

Comment (kevinz000) [inline code/datums/weather/weather_types.dm]: SHIT.

Comment (kevinz000) [inline code/datums/weather/weather_types.dm]: Oh well that sucks. >Weather doesn't have an innate cooldown between impact calls

**Comment by PKPenguin321** [comment]
@Shadowlight213 
>True. I can increase the startup phase delay.
I might use the blob alarm sound or whatever for the winding up phase as well.

Just do this and focus on the overlay, we need a competing PR so this garbage doesnt get merged

**Comment by KorPhaeron** [comment]
Well this PR is entirely non functional and Shadowlight said he's gonna change things himself, we can revisit this later if we need to.

Dealing tox damage per tick (like ash storms) with only a chance of doing rads/mutation might be better than chainstun to death for anyone who ever sticks their head out though

**Pull request closed without merging**